client/src/app/containers/Game/PlayerHand.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Grid from '@material-ui/core/Grid';
+import GridList from '@material-ui/core/GridList';
+import { nanoid } from '@reduxjs/toolkit';
+
+import { Card, PeggedCard, Phase } from './models';
+import PlayingCard from './PlayingCard';
+
+const showOpponentsHand = (phase: Phase) => phase !== 'Deal';
+
+interface Props {
+    hand: Card[];
+    phase: Phase;
+    side?: boolean;
```

Comment:
:mag: :book: what does the `?` mean? Does it mean "optional"?

Reply:
Yes; it's syntactic sugar and equivalent to saying `side: boolean | undefined;`